From: Colin Watson <cjwatson@debian.org>
Date: Mon, 7 Apr 2025 10:15:15 +0100
Subject: Adjust test for rust-url 2.5.4

Bug-Debian: https://bugs.debian.org/1102283
Last-Update: 2025-04-07
---
 tests/test_networks.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/test_networks.py b/tests/test_networks.py
index 34e1c07..ecc9aa1 100644
--- a/tests/test_networks.py
+++ b/tests/test_networks.py
@@ -128,7 +128,7 @@ def test_any_url_success(value):
         (
             'https://example.org more',
             'url_parsing',
-            'Input should be a valid URL, invalid domain character',
+            'Input should be a valid URL, invalid international domain name',
         ),
         ('$https://example.org', 'url_parsing', 'Input should be a valid URL, relative URL without a base'),
         ('../icons/logo.gif', 'url_parsing', 'Input should be a valid URL, relative URL without a base'),
